app/grandchallenge/reader_studies/models.py

```diff
@@ -1508,3 +1552,31 @@ def delete(self):
 
     class Meta(RequestBase.Meta):
         unique_together = (("reader_study", "user"),)
+
+
+class AnswerWidgetKindChoices(models.TextChoices):
+    ACCEPT_REJECT = "ACCEPT_REJECT", "Accept/Reject Findings"
+
+
+class AnswerWidget(models.Model):
+
+    kind = models.CharField(
+        max_length=255, choices=AnswerWidgetKindChoices.choices, editable=False
+    )
+
+    def __str__(self):
+        return self.get_kind_display()
+
+    @staticmethod
+    def supported_answer_types(self):
+        raise NotImplementedError("Subclasses must implement this method.")
+
+
+class AcceptRejectFindingsWidget(AnswerWidget):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.kind = AnswerWidgetKindChoices.ACCEPT_REJECT
+
+    @staticmethod
+    def supported_answer_types():
+        return AnswerType.get_multiple_annotation_types()
```

Comment:
Probably better to put each answer type in here directly as this will be closely tied to what is actually implemented in CIrRUS.

Reply:
Yes, I think a future developer would naturally extend `get_multiple_annotation_types` without realising that it would implicitly mean that CIRRUS would support it. The widget should define what CIRRUS supports.

Reply:
Good point.